lab5.py

```diff
@@ -0,0 +1,98 @@
+"""
+This is lab5
+"""
+class Smartphone:
+    """
+    This is class where write my parameter
+    """
+    def __init__(self, price, model, phone_number, memory, battery):
+        self.price = price
+        self.model = model
+        self.phone_number = phone_number
+        self.memory = memory
+        self.battery = battery
+
+    def __str__(self):
+        """
+        This is method str
+        """
+        return f"{self.price}, {self.model}, {self.phone_number}, {self.memory}, {self.battery}"
+
+    def see_price(self):
+        """
+        this function return my phone price
+        """
+        return f"Phone price: {self.price}"
+
+
+class PhoneStore:
+    """
+    this class where my phones
+    """
+
+
+    @staticmethod
+    def add_phone(smartphone):
+        This function append smartphone to inventory
+        """
+        self.inventory.append(smartphone)
+
```

Comment:
you've missed to close your comment here, so code will not work and output many errors

```suggestion
"""
```